handlers/check.go

```diff
@@ -204,31 +172,6 @@ func updateHighScores(mb *bolt.Bucket, resp checksResp, repo string) error {
 	return nil
 }
 
-func updateStats(mb *bolt.Bucket, resp checksResp, repo string, oldScore *float64) error {
```

Comment:
actually, not sure if we want to remove this, we're not displaying the data anywhere though afaik